apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -290,14 +294,14 @@ const CurriculumCatalog = ({curriculaData, isEnglish}) => {
             handleClearAll={() => handleClearAllOfFilter(filterKey)}
           />
         ))}
-        <button
+        <Button
           id="clear-filters"
-          type="button"
           className={style.catalogClearFiltersButton}
```

Comment:
Same question here –– not convinced these styles are doing anything at this point